src/eduid_webapp/actions/tests/test_app.py

```diff
@@ -47,6 +50,15 @@ class ActionsTestCase: pass
 
 class ActionsTests(ActionsTestCase):
 
+    def update_actions_config(self, config):
+        config['TOKEN_LOGIN_SHARED_KEY'] = config['TOKEN_LOGIN_SHARED_KEY'][:nacl.secret.SecretBox.KEY_SIZE]
+        if len(config['TOKEN_LOGIN_SHARED_KEY']) < 32:
+            config['TOKEN_LOGIN_SHARED_KEY'] += (32 - len(config['TOKEN_LOGIN_SHARED_KEY'])) * '0'
```

Comment:
This looks like it might cause one to accidentally use 32x'0' as the shared key.

Also, use nacl.secret.SecretBox.KEY_SIZE instead of hard coding in all places, not just the first =)

Reply:
Yes sorry for the hardcoded 32, fixing it.

Regarding the 32x'0', this is just the test case, do you mean that someone might take it as sort of documentation and copy the 32 0s over to their live config? The reason I adjusted the length is that the setting was coming from a test case defined in another package (eduid_action) that was not fixed for that issue.

Reply:
I mean that I agree with removing the code to adjust the length of the key, but I'd do it after the egg depends on a newer version of eduid-action with a correct key length. Do you agree?

Reply:
Oh, in a test case I don't mind a default of 32 x '0' =).

I don't understand the issue with the key length, so do what you think is best there.